### DISCUSSION TITLE
leetcode-js/

### DISCUSSION THREAD

**Discussion by giscus** [2xiao/leetcode-js#23] [Announcements]
# leetcode-js/

包含 LeetCode、《剑指 Offer》、《剑指 Offer II》等 1000+ 道算法题的详细题解。 书中的算法全部使用 JavaScript 语言实现，前端工程师友好。 欢迎「Star ⭐️ 」 和 「Fork」，这是对我最大的鼓励和支持。 电子书：https://2xiao.github.io/leetcode-js (https://2x...

https://2xiao.github.io/leetcode-js/

**Comment by mushroom2163** [comment]
👍赞一个
